### DISCUSSION TITLE
Progresso de Vacinação, com estimativa de 70%

### DISCUSSION THREAD

**Discussion by diogotcorreia** [dssg-pt/covid19pt-data#851] [Montra de Projectos]
![2021-05-30_18-09](https://user-images.githubusercontent.com/7467891/120116627-ed87b280-c180-11eb-8f89-da64e008d48a.png)

Site disponível em https://covid.diogotc.com

**Comment by ishouldbedany** [comment]
Muito bom aspecto, parabéns! Também vi o post no /r/Portugal, obrigado pelo _shoutout_!

Acho que seria interessante haver uma nota metodológica acerca do método de cálculo dos 70%. Presumo que seja por estimativa linear mediante a velocidade dos últimos dias, mas o valor real provavelmente diferirá dado os diferentes intervalos entre as 2 doses de cada vacina (para as que incluem 2 doses).

**Reply by davipt** [reply]
Há tentativas de previsões no tweet_vacinas e no notebooks/temp/temp_vacinas e _ecdc, mas penso que há demasiadas variáveis desconhecidas para poder fazer uma previsão para lá das próximas quatro semanas. Se quiseres podemos trocar umas ideias e ver todas as condicionantes que já passei. 

Há uma cópia de um dos notebooks em https://colab.research.google.com/drive/1u8G_HDj9Yh_wcw3jLhvj68QdfI6w2sbS
3 milhões lá para dia 20 e picos de junho :)
